localCFBtesting.postman_environment.json

```diff
@@ -0,0 +1,94 @@
+{
+	"id": "2ec591cf-84b3-47cc-a0bc-f8f627ed96ba",
+	"name": "localCFBtesting",
+	"values": [
+		{
+			"key": "baseUrl",
+			"value": "http://localhost:3000",
+			"enabled": true
+		},
+		{
+			"key": "testAdminUsername",
+			"value": "homer.simpson@sfpp.com",
+			"enabled": true
+		},
+		{
+			"key": "testAdminPassword",
+			"value": "donuts",
+			"enabled": true
+		},
+		{
+			"key": "badUsername",
+			"value": "mom@sfpp.com",
+			"enabled": true
+		},
+		{
+			"key": "malformedUsername",
+			"value": "momsfpp.com",
+			"enabled": true
+		},
+		{
+			"key": "badToken",
+			"value": "Imabadtoken",
+			"enabled": true
+		},
+		{
+			"key": "token",
+			"value": "Too many requests, please try again later.",
+			"enabled": true
+		},
+		{
+			"key": "firstUserEmail",
+			"value": "homer.simpson@sfpp.com",
+			"enabled": true
+		},
+		{
+			"key": "newUserEmail",
+			"value": "marge.simpson@sfpp.com",
+			"enabled": true
+		},
+		{
+			"key": "firstContactId",
+			"value": "3ae16f77-848b-49ab-9026-1cb68ab5be3e",
+			"enabled": true
+		},
+		{
+			"key": "secondContactId",
+			"value": "7d788b02-60f7-4398-b5d8-8ab4fd2e6670",
+			"enabled": true
+		},
+		{
+			"key": "newContactId",
+			"value": "4d783684-8f50-42cc-8d5e-04f9caf6c704",
+			"enabled": true
+		},
+		{
+			"key": "firstEntityId",
+			"value": "23dfff7a-d501-4ee5-9a59-4ceaa76ad5e9",
+			"enabled": true
+		},
+		{
+			"key": "secondEntityId",
+			"value": "69c1c53a-11da-4e88-a9b5-b623f46cf0b1",
+			"enabled": true
+		},
+		{
+			"key": "newEntityId",
+			"value": "5b67b241-5378-44b7-b833-713209c79138",
+			"enabled": true
+		},
+		{
+			"key": "newTestContactId",
+			"value": "04d9ff55-aa91-40e1-bb33-cf4ab3ecb2dc",
+			"enabled": true
+		},
+		{
+			"key": "newTestEntityId",
+			"value": "929cc48b-4e03-4bcd-a270-aae266d6982d",
```

Comment:
If these values are randomly generated on seeding, will putting them here work in a new environment?